Process.h

```diff
@@ -60,6 +62,7 @@ typedef struct ProcessPidColumn_ {
 
 typedef struct Process_ {
    Object super;
+   struct ProcessList_* processList;
```

Comment:
As this is non-owning it should probably be `const`.

Reply:
Fixed